ref/general/federated-registries.adoc

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: Configuring an LDAP user registry for authentication and authorization
+:seo-description: You can configure an LDAP user registry to manage authentication and authorization for your applications that run on Open Liberty.
+:page-layout: general-reference
+:page-type: general
+= Federating user registries
+
+You can combine multiple user registries into a single federated registry to manage authentication and authorization for your application.
+
+In cases where user and group information is spread across multiple registries, federation enables Open Liberty to use this distributed user information in a unified manner with continuous store of information. Different types of registries can be federated so that information from LDAP, basic, and custom user registries can be combined into a common registry to manage application security. However, user registries that are configured with the `quickStartSecurity` element cannot be federated with other registries.
+
+== Federation of LDAP registries
+
+LDAP registries are federated by default. If you configure more than one LDAP registry in your `server.xml` file, then the registries are automatically federated with no additional configuration required. These registries are federated by default regardless of whether they are separate LDAP registries or two subtrees of the same LDAP registry.
```

Comment:
Change,
"These registries are federated by default regardless of whether they are separate LDAP registries or two subtrees of the same LDAP registry."
to,
"This federation happens by default regardless of whether the LDAP registries are separate, or two subtrees of the same LDAP registry."